markdown/dev/guides/patterns/stacks/en.md

```diff
@@ -3,7 +3,7 @@ title: Stacks
 order: 110
 ---
 
-Stacks come into play when laying out a pattern.
+[Stacks](/reference/api/stack) come into play when laying out a pattern.
 The FreeSewing core library, by default, will handle the layout of a pattern
 for you by placing all parts next to each other in as small a space as
 possible.
```

Comment:
Let's change this to "by placing all parts into stacks, and arranging the stacks in as small a space as possible without overlapping". 

The point is that parts in the same stack will overlap, but stacks shouldn't because they are the boxes used by the layout engine

Reply:
Added.